data/styles/clear/include/Basemap.mapcss

```diff
@@ -287,10 +290,9 @@ line|z15-[waterway=stream][intermittent=yes]
 node|z16-[waterway=lock_gate],
 {icon-image: dot-m.svg;}
 
-line|z17-[waterway=ditch],
-line|z17-[barrier=ditch],
-line|z17-[waterway=drain],
-{width: 1.8;opacity: 1;color: @river;dashes: 0.9,0.9;}
+line|z14-[waterway=ditch],
+line|z14-[waterway=drain],
+{width: 1.8;opacity: 1;color: @water_bad;dashes: 12,2;}
```

Comment:
I see you've raised visibility significantly. Very good for outdoors, but won't it create issues in cities?
Either test extensively or raise 1 level max at a time :)

Also, could you please add a screenshot of the changed pattern and color too?

Reply:
i don't think it'll be an issue, streams render from z14 and ditch/drain just as a line are very faint, but are still useful to see